From: Michael Biebl <biebl@debian.org>
Date: Sat, 30 Aug 2014 02:01:51 +0200
Subject: libudev: fix symbol version for udev_queue_flush() and
 udev_queue_get_fd()

Those symbols were introduced in commit
14cb733684d3c3f50d088a3a370ddf8e8894dfa4 and released in v215.

(cherry picked from commit 015419c0dfd96bef9401c868137711ca78cce652)
---
 src/libudev/libudev.sym | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/libudev/libudev.sym b/src/libudev/libudev.sym
index 95cd1c7..76726fc 100644
--- a/src/libudev/libudev.sym
+++ b/src/libudev/libudev.sym
@@ -82,8 +82,6 @@ global:
         udev_queue_get_udev;
         udev_queue_get_udev_is_active;
         udev_queue_get_udev_seqnum;
-        udev_queue_get_fd;
-        udev_queue_flush;
         udev_queue_new;
         udev_queue_ref;
         udev_queue_unref;
@@ -114,3 +112,9 @@ LIBUDEV_199 {
 global:
         udev_device_set_sysattr_value;
 } LIBUDEV_196;
+
+LIBUDEV_215 {
+global:
+        udev_queue_flush;
+        udev_queue_get_fd;
+} LIBUDEV_199;
